src/odemis/driver/xt_client.py

```diff
@@ -376,6 +386,15 @@ def move_stage(self, position: Dict[str, float], rel: bool = False) -> None:
             self.server._pyroClaimOwnership()
             self.server.move_stage(position, rel)
 
+    def set_raw_coordinate_system(self, raw: bool = True):
```

Comment:
Even if you don't use it now, can you add the `get_raw_coordinate_system()` function so that we know it exists?

Reply:
@tepals the reason of adding this function is explained here

Reply:
I think what Éric meant (but correct me if I'm wrong) is that since you have the function on the xtadapter side, you might as well implement it here. Now you just have it implemented as an empty function, while you could also implement the working call to the xtadapter. So not just do `pass` but actually call the function.

Reply:
Yes, I confirm this interpretation from Thera :-)
You have to actually implement `get_raw_coordinate_system()`: call the xtadapter's function and return the returned value)